apps/console/src/features/groups/pages/groups.tsx

```diff
@@ -338,13 +338,7 @@ const GroupsPage: FunctionComponent<any> = (): ReactElement => {
                     </RootOnlyComponent>
                 ) }
                 showPagination={ paginatedGroups?.length > 0  }
-                showTopActionPanel={
-                    !isGroupsListRequestLoading
-                    && !(!searchQuery
-                        && !groupsError
-                        && userStoreOptions?.length < 3
-                        && (!paginatedGroups || paginatedGroups?.length <= 0))
```

Comment:
@dasuni-30 Do we really need to check these in order to enable the top search bar?

Reply:
Let's check if this affects the cloud product